internal/provider/models/translate_ssh_tunnel.go

```diff
@@ -6,22 +6,28 @@ import (
 	"terraform-provider-artie/internal/artieclient"
 )
 
-func SSHTunnelAPIToResourceModel(apiModel artieclient.SSHTunnel, resourceModel *SSHTunnelResourceModel) {
-	resourceModel.UUID = types.StringValue(apiModel.UUID.String())
-	resourceModel.Name = types.StringValue(apiModel.Name)
-	resourceModel.Host = types.StringValue(apiModel.Host)
-	resourceModel.Port = types.Int32Value(apiModel.Port)
-	resourceModel.Username = types.StringValue(apiModel.Username)
-	resourceModel.PublicKey = types.StringValue(apiModel.PublicKey)
+func (s *SSHTunnelResourceModel) FillFromAPIModel(apiModel artieclient.SSHTunnel) {
```

Comment:
`FillFromAPIModel` feels a little clunky to me. What about `UpdateFromAPIModel`?